### PR TITLE
New swap copy + icons

### PR DESCRIPTION
**Pull request by grod220**
Closes: https://github.com/penumbra-zone/web/issues/532

Noticed the icons did not really line up with the page content. Snagged a few new ones.

Screenshot 2024-02-16 at 5 44 57 PM
Screenshot 2024-02-16 at 5 46 02 PM
Screenshot 2024-02-16 at 5 46 05 PM
Screenshot 2024-02-16 at 5 46 09 PM
Screenshot 2024-02-16 at 5 46 13 PM

### CODE REVIEW

File: apps/webapp/src/components/header/constants.tsx [hunk 1]
```diff
@@ -33,12 +32,6 @@ export const headerLinks: HeaderLink[] = [
     active: true,
     mobileIcon: <SwapIcon />,
   },
-  {
-    href: PagePath.POOLS,
-    label: 'Pools',
-    active: false,
-    mobileIcon: <DragHandleDotsIcon />,
-  },
```
Comment (grod220) [inline apps/webapp/src/components/header/constants.tsx:41]: minifront won't be supporting liquidity provisioning

File: apps/webapp/src/components/shared/edu-panels/content.ts [hunk 2]
```diff
@@ -21,6 +22,8 @@ export const eduPanelContent: Record<EduPanel, string> = {
     'Every Penumbra account has a stable default address and many one-time IBC deposit addresses.  All addresses for the same account deposit to the same pool of funds.  Use a freshly generated IBC deposit address to preserve privacy when sending funds from a transparent chain to Penumbra.',
   [EduPanel.IBC_WITHDRAW]:
     'IBC to a connected chain. Note that if the chain is a transparent chain, the transaction will be visible to others.',
+  [EduPanel.SWAP]:
+    'Shielded swaps between any kind of cryptoasset, with sealed-bid, batch pricing and no frontrunning. Only the batch totals are revealed, providing long-term privacy. Penumbra has no MEV, because transactions do not leak data about user activity.',
```
Comment (grod220) [inline apps/webapp/src/components/shared/edu-panels/content.ts:26]: Taken from https://penumbra.zone/technology/swap/

File: apps/extension/public/manifest.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 3,
   "name": "Penumbra wallet",
-  "version": "4.0.0",
+  "version": "4.1.0",
```
Comment (grod220) [inline apps/extension/public/manifest.json:4]: Prepping for: https://github.com/penumbra-zone/web/issues/534

File: apps/webapp/src/components/header/constants.tsx (hunk 1 above)
Comment (jessepinho) [reply apps/webapp/src/components/header/constants.tsx:41]: What's minifront? 😬

**Review by jessepinho — APPROVED** [review]

File: apps/webapp/src/components/header/constants.tsx (hunk 1 above)
Comment (grod220) [reply apps/webapp/src/components/header/constants.tsx:41]: "mini frontend" is what we wanted to rename "web app" https://discord.com/channels/824484045370818580/1044457038886998137/1200144643321516032

**Pull request merged**